src/gd.c

```diff
@@ -2339,8 +2342,8 @@ BGD_DECLARE(void) gdImageRectangle (gdImagePtr im, int x1, int y1, int x2, int y
 	}
 }
 
-BGD_DECLARE(void) gdImageFilledRectangle (gdImagePtr im, int x1, int y1, int x2, int y2,
-		int color)
+static void _gdImageFilledRectangle (gdImagePtr im, int x1, int y1, int x2, int y2,
+		int color, int horizontal)
```

Comment:
Oh Right, I missed that the new functions are also used for generic rectangles.

You are right too about H and V as I can see needs for that in other area where styles or patterns are used.